src/app.js

```diff
@@ -418,10 +421,18 @@ var run = function () {
     }
   })
 
+  // convert old browser-solidity
```

Comment:
Realized that I still want to keep there the removal of the key, so for the moment this can stay. In #377 it will only delete key.